docs/awesome-list.md

```diff
@@ -16,6 +16,9 @@ HiFiGAN vocoder using two discriminators to map mel-spectrograms to audio.
 **nanoGPT**: [GitHub](https://github.com/TugdualKerjan/nanoEQXGPT)  
 Port of nanoGPT, a minimalist implementation of a Generative Pre-trained Transformer.
 
+**equinox-openpi** [Github](https://github.com/vyeevani/equinox-openpi)
```

Comment:
I think you're missing the double space at the end of this line, that is needed to introduce a newline here.